R/subscript.R

```diff
@@ -126,7 +131,6 @@ vec_as_subscript2_result <- function(i,
   if (typeof(i) == "logical") {
```

Comment:
Can you remove this section now? That's what I was hoping for

Reply:
Thanks, now simplified.